content/install/server/database.md

```diff
@@ -22,8 +22,8 @@ services:
   drone-server:
     image: drone/drone:{{% version %}}
     environment:
-+     DRONE_DATABASE_DRIVER: mysql
-+     DRONE_DATABASE_DATASOURCE: root:password@tcp(1.2.3.4:3306)/drone?parseTime=true
++     - DRONE_DATABASE_DRIVER:mysql
```

Comment:
```
- DRONE_DATABASE_DRIVER=mysql
```

and the same for the other array lines, it should be `=`and not `:`

Reply:
As mentioned in the very first review, using `=` instead of `:` throws the following message in repetition.

```
drone-agent_1   | INFO: <redacted_date> <redacted_time> grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: Error while dialing dial tcp <redacted_ip>:<redacted_port>: getsockopt: connection refused"; Reconnecting to {drone-server:<redacted_port> <nil>}
drone-agent_1   | <redacted_date> <redacted_time> grpc error: done(): code: Unavailable: rpc error: code = Unavailable desc = grpc: the connection is unavailable
```